tests/acceptance/change.spec.js

```diff
@@ -0,0 +1,176 @@
+        expect(contents).to.equal(expectedContents);
+      });
+  })
+
+
+});
+
+describe('RemoveChange: remove string from the source code', () => {
+  let sourcePath = 'src/app/my-component';
+  beforeEach(() => {
+    let mockDrive = {
+      'src/app/my-component': {
+        'component-file.txt': 'import * as foo from "./bar"'
+      }
+
+    };
+    mockFs(mockDrive);
+  });
+
+  afterEach(() => {
+    mockFs.restore();
+  });
+
+  it('should remove text from the source code', () => {
+    let sourceFile = path.join(sourcePath, 'component-file.txt');
+    expect(fs.existsSync(sourceFile)).to.equal(true);
+    let changeInstance = new change.RemoveChange(sourceFile, 9, 'as foo');
+    return changeInstance
+      .apply()
+      .then(() => {
+        return fsPromise.readWithPromise(sourceFile);
+      }).then(contents => {
+        let expectedContents = 'import *  from "./bar"';
+        expect(contents).to.equal(expectedContents);
+      });
+  })
+
+  it('should remove text from the source code', () => {
+    let sourceFile = path.join(sourcePath, 'component-file.txt');
+    expect(fs.existsSync(sourceFile)).to.equal(true);
+    let changeInstance = new change.RemoveChange(sourceFile, 9, '');
+    return changeInstance
+      .apply()
+      .then(() => {
+        return fsPromise.readWithPromise(sourceFile);
+      }).then(contents => {
+        let expectedContents = 'import * as foo from "./bar"';
+        expect(contents).to.equal(expectedContents);
+      });
+  })
+
+});
+
+describe('ReplaceChange: replace string in the source code', () => {
+
+  let sourcePath = 'src/app/my-component';
+  beforeEach(() => {
+    let mockDrive = {
+      'src/app/my-component': {
+        'component-file.txt': 'import * as foo from "./bar"'
+      }
+
+    };
+    mockFs(mockDrive);
+    sourcePath = path.join('src/app/my-component');
+  });
+
+  afterEach(() => {
+    mockFs.restore();
+  });
+
+  it('should replace text in the source code', () => {
+    let sourceFile = path.join(sourcePath, 'component-file.txt');
+    expect(fs.existsSync(sourceFile)).to.equal(true);
+    let changeInstance = new change.ReplaceChange(sourceFile, 7, '* as foo', '{ fooComponent }');
+    return changeInstance
+      .apply()
+      .then(() => {
+        return fsPromise.readWithPromise(sourceFile);
+      }).then(contents => {
+        let expectedContents = 'import { fooComponent } from "./bar"';
+        expect(contents).to.equal(expectedContents);
+      });
+  })
+
+  it('should replace text in the source code', () => {
+    let sourceFile = path.join(sourcePath, 'component-file.txt');
+    expect(fs.existsSync(sourceFile)).to.equal(true);
+    let changeInstance = new change.ReplaceChange(sourceFile, 7, '', '{ fooComponent }');
+    return changeInstance
+      .apply()
+      .then(() => {
+        return fsPromise.readWithPromise(sourceFile);
+      }).then(contents => {
+        let expectedContents = 'import { fooComponent }* as foo from "./bar"';
```

Comment:
Maybe add to the import path then: `from "./bar/baz"`.

Not that the code is wrong, but it is a little confusing to see the test expecting invalid syntax.